caluma/workflow/tests/test_work_item.py

```diff
@@ -287,6 +334,57 @@ def test_complete_work_item_with_next_multiple_tasks(
     ) == {task_next_1.pk, task_next_2.pk}
 
 
+@pytest.mark.parametrize(
+    "work_item__status,work_item__child_case,task__type",
+    [(models.WorkItem.STATUS_READY, None, models.Task.TYPE_SIMPLE)],
+)
+def test_complete_work_item_with_next_multiple_instance_task(
+    db,
+    case,
+    work_item,
+    task,
+    task_factory,
+    task_flow_factory,
+    workflow,
+    schema_executor,
+):
+    task_next = task_factory.create(
+        is_multiple_instance=True, address_groups=["group1", "group2", "group3"]
+    )
+    task_flow = task_flow_factory(task=task, workflow=workflow)
+    task_flow.flow.next = f"['{task_next.slug}']|task"
+    task_flow.flow.save()
+
+    query = """
+        mutation CompleteWorkItem($input: CompleteWorkItemInput!) {
+          completeWorkItem(input: $input) {
+            workItem {
+              status
+              case {
+                status
+                workItems {
+                  edges {
+                    node {
+                      status
+                    }
+                  }
+                }
+              }
+            }
+            clientMutationId
+          }
+        }
+    """
+
+    inp = {"input": {"id": work_item.pk}}
+    result = schema_executor(query, variables=inp)
+
+    assert not result.errors
+    assert case.work_items.count() == 4
```

Comment:
this line is not quite clear why... better to check for completed (1) and ready work items (3)?

Reply:
Got point, done